packages/template-retail-react-app/app/constants.js

```diff
@@ -29,8 +29,10 @@ export const HOME_SHOP_PRODUCTS_CATEGORY_ID = 'newarrivals'
 export const HOME_SHOP_PRODUCTS_LIMIT = 10
 
 // Constants for menu
-export const CAT_MENU_DEFAULT_NAV_DEPTH = 1
+export const CAT_MENU_DEFAULT_NAV_SSR_DEPTH = 1
+export const CAT_MENU_DEFAULT_NAV_CLIENT_DEPTH = 3
```

Comment:
```suggestion
export const CAT_MENU_DEFAULT_NAV_DEPTH_SSR = 1
export const CAT_MENU_DEFAULT_NAV_DEPTH_CLIENT = 3
```
Feels weird for the difference to not be the first or last word. 🤷